### ISSUE TITLE
Question on semantic loss of CycleGANSemanticModel

### ISSUE THREAD

**Issue by yklInverted** [junyanz/pytorch-CycleGAN-and-pix2pix#1508]
Hi, I got a bit confused about the implementation of the semantic loss in the CycleGANSemanticModel. In your CyCADA paper, the semantic consistency loss is computed using a pretrained model fs. However, in this code I found that the semantic loss is directly computed using the target model ft. I just wonder why the implementation differs and how will this influence the result.
